libraries/Ticker/Ticker.h

```diff
@@ -96,10 +107,14 @@ class Ticker
 
 protected:	
 	void _attach_ms(uint32_t milliseconds, bool repeat, callback_with_arg_t callback, uint32_t arg);
+	static void internalCallback (void* arg);
 
 
 protected:
 	ETSTimer* _timer;
+	TickerFunction internalTicker = nullptr;
```

Comment:
For consistency with the rest of this file, please use snake_case for member and function names, and prefix the member names with an underscore. Also, perhaps `_callback` would be a better name for this member? `internalTicker` makes it look like there is an instance of `Ticker` here.